thrift/arg2/kv_iterator.go

```diff
@@ -0,0 +1,87 @@
+package arg2
```

Comment:
should we add package docs that this is not a supported stable API, but is internal for relays?